hmpps-sqs-spring-boot-autoconfigure/src/main/kotlin/uk/gov/justice/hmpps/sqs/HmppsQueueService.kt

```diff
@@ -28,71 +33,75 @@ open class HmppsQueueService(
     private val gson = GsonBuilder().setObjectToNumberStrategy(ToNumberPolicy.LONG_OR_DOUBLE).create()
   }
 
-  private val hmppsQueues: List<HmppsQueue> = hmppsQueueFactory.createHmppsQueues(hmppsSqsProperties, hmppsTopicService.hmppsTopics, hmppsTopicService.hmppsAsyncTopics)
+  private val hmppsQueues: List<HmppsQueue> = hmppsQueueFactory.createHmppsQueues(hmppsSqsProperties, hmppsTopicService.hmppsTopics)
 
   open fun findByQueueId(queueId: String) = hmppsQueues.associateBy { it.id }.getOrDefault(queueId, null)
   open fun findByQueueName(queueName: String) = hmppsQueues.associateBy { it.queueName }.getOrDefault(queueName, null)
   open fun findByDlqName(dlqName: String) = hmppsQueues.associateBy { it.dlqName }.getOrDefault(dlqName, null)
 
-  open fun retryDlqMessages(request: RetryDlqRequest): RetryDlqResult =
+  open suspend fun retryDlqMessages(request: RetryDlqRequest): RetryDlqResult =
```

Comment:
I think we still need 2 copies of the service - one for the reactive API with suspend functions and 1 for the non-reactive API. However the old name of `HmppsAsyncQueueService` is probably wrong and confusing - maybe better to call it `HmppsReactiveQueueService`?

Reply:
Hmmm now I think about it maybe we do just need 1 copy of the service with suspend functions - the non-reactive API can then just wrapp them all in runBlocking as the current version does.